docs/open_source/index.rst

```diff
@@ -0,0 +1,10 @@
+Open Source
+===============
+
+.. toctree::
```

Comment:
Same as for integrations :

Should we use :hidden: here, and use cards instead ?

Reply:
done